[core]/es_extended/client/functions.lua

```diff
@@ -420,20 +402,19 @@ function ESX.Game.GetObjects() -- Leave the function for compatibility
 end
 
 function ESX.Game.GetPeds(onlyOtherPeds)
-    local myPed, pool = ESX.PlayerData.ped, GetGamePool('CPed')
-
-    if not onlyOtherPeds then
-        return pool
-    end
-
-    local peds = {}
-    for i = 1, #pool do
-        if pool[i] ~= myPed then
-            peds[#peds + 1] = pool[i]
+    local pool = GetGamePool('CPed')
```

Comment:
I don't think it's lucky that the full pool is returned by default. Because then compatibility is broken because the default is to return an empty table, which is what the old method did. Now it will always return the pool. So it won't be backwards compatible.